ap-java-sdk/src/test/java/com/alternativepayments/apimocks/CustomerApiMock.java

```diff
@@ -15,44 +26,151 @@ private CustomerApiMock() {
+                        .withBody("{\n"
+                            + "  \"customers\": [\n"
+                            + "  {\n"
+                            + "  \"id\": \"cus_7f0724f3b1d745d49\",\n"
+                            + "  \"mode\": \"Live\",\n"
+                            + "  \"firstName\": \"Jane\",\n"
+                            + "  \"lastName\": \"Doe\",\n"
+                            + "  \"email\": \"jane@doe.com\",\n"
+                            + "  \"country\": \"DE\",\n"
                             + "  \"created\": \"2012-04-23T18:25:43.511Z\"\n"
                             + "  },\n"
                             + "  {\n"
                             + "  \"id\": \"cus_7f0724f3b1d745d49\",\n"
                             + "  \"firstName\": \"John\",\n"
                             + "  \"lastName\": \"Doe\",\n"
+                            + "  \"mode\": \"Live\",\n"
+                            + "  \"email\": \"john@doe.com\",\n"
+                            + "  \"address\": \"Rutledge Ave 409\",\n"
+                            + "  \"city\": \"Folsom\",\n"
+                            + "  \"zip\": \"19033\",\n"
+                            + "  \"country\": \"US\",\n"
+                            + "  \"state\": \"PA\",\n"
+                            + "  \"phone\": \"55555555555\",\n"
+                            + "  \"created\": \"2012-04-23T18:25:43.511Z\"\n"
+                            + "  },\n"
+                            + "  {\n"
+                            + "  \"id\": \"cus_15b154da233sxcvby\",\n"
+                            + "  \"firstName\": \"John\",\n"
+                            + "  \"lastName\": \"Wayne\",\n"
+                            + "  \"mode\": \"Live\",\n"
+                            + "  \"email\": \"wayne.john@doe.com\",\n"
+                            + "  \"address\": \"Wild wild west\",\n"
+                            + "  \"city\": \"Texas\",\n"
+                            + "  \"zip\": \"19033\",\n"
+                            + "  \"country\": \"US\",\n"
+                            + "  \"state\": \"Texas\",\n"
                             + "  \"created\": \"2012-04-23T18:25:43.511Z\"\n"
                             + "  }\n"
                             + "  ],\n"
                             + "  \"pagination\": {\n"
                             + "  \"offset\": 10,\n"
-                            + "  \"limit\": 2,\n"
+                            + "  \"limit\": 3,\n"
                             + "  \"count\": 2504\n"
                             + "  }\n"
                             + "}")));
     }
+
+    public static void expectPost(final Customer customer) {
+        try {
+            WireMock.stubFor(
+                    post(urlMatching("/api/customers/"))
+                    .withHeader("Authorization", containing(""))
+                    .withRequestBody(containing(new ObjectMapperProvider()
```

Comment:
@nbozic  I'm pretty sure there is a prettier, more readable way for creating json body for post request as this is just ugly?